test/specs/test.e2e.ts

```diff
@@ -78,16 +78,26 @@ describe('Verify user can add product to cart', () => {
         const cartButton = $('[data-test="nav-cart"]')
         console.log('cartButton');
         await cartButton.click()
-        console.log('cartButton click');
-        
+        //console.log('cartButton click');
```

Comment:
You can add https://practicesoftwaretesting.com to wdio.conf.ts file as baseURL https://webdriver.io/docs/configurationfile/